### PR TITLE
feat: add ctrl+k combination shortcut for search

### PR DESCRIPTION
**Pull request by anuraglol**
added `ctrl + k`/`cmd + k` combination to trigger the search modal in the `fonts` page

### CODE REVIEW

**Comment by ayuhito** [comment]
Oops, I still haven't had time to add lint-staged but the build is failing due to some linting issues. Could you resolve that manually for now?

**Comment by anuraglol** [comment]
> Oops, I still haven't had time to add lint-staged but the build is failing due to some linting issues. Could you resolve that manually for now?

hey, when i run `yarn lint` it runs a command, says its done, but not a single file is changed in the editor, which is weird

**Comment by ayuhito** [comment]
Ah, try running `next lint --fix` and `yarn format` in the websites directory.

**Comment by anuraglol** [comment]
> Ah, try running `next lint --fix` and `yarn format` in the websites directory.

hey, ran the command, still failed, some kind of new error?

**Comment by anuraglol** [comment]
> Ah, try running `next lint --fix` and `yarn format` in the websites directory.

hey, ran the command, still failed, some kind of new error?

**Comment by ayuhito** [comment]
Actually, it’s fine. Thank you for your contribution! I’ll test this branch manually later today, merge your code and resolve the lint errors myself because this is a misconfiguration on my end. I’m used to my editor autofixing so I never actually added the right scripts for linting into this repo itself. Sorry!

File: website/src/components/Search/Modal.tsx [hunk 1]
```diff
@@ -40,7 +58,15 @@ export const SearchModal = () => {
         aria-label="Search fonts"
         role="search"
       >
-        Search
+        <Box>
+          <SearchIcon ml="-5px" mr="10px" />
+          Search
+        </Box>
+
+        <Box>
```
Comment (ayuhito) [inline website/src/components/Search/Modal.tsx]: I think it'd be good to hide this when the website switches to mobile view. 

Also, it does feel vertically unaligned. Am I the only one seeing that?

**Review by ayuhito — CHANGES_REQUESTED** [review]

**Comment by ayuhito** [comment]
Alrighty, I took a look at the code. While we iron out the code review changes, could you also pull the main branch into this branch and then run yarn? I added lint-staged and fixed some linting configs so everything should automatically fix itself (I also realised your changes also had a false positive error from a lint rule too). 

You can also run `yarn lint:fix` in the website directory if things don't auto-fix.

**Comment by ayuhito** [comment]
@kr-anurag, I might've unintentionally done something really bad to the repository. Please don't pull or push anything for the time being.

**Comment by anuraglol** [comment]
> @kr-anurag, I might've unintentionally done something really bad to the repository. Please don't pull or push anything for the time being.

hey! umm okay!

**Comment by ayuhito** [comment]
@kr-anurag, I'll have to apologise for interrupting your PR mid-way. This is definitely not a regular occurrence.

Just for context on what happened to your PR, #456 would explain the situation. Essentially we had to rewrite the entire Git history which is a destructive process, but it was somewhat needed for this repo. Ideally, I would've wanted to do this AFTER you submitted your PR, but I accidentally pushed to the main repository instead of a test repository 😭 

To fix your PR, you could either make a **fresh clone** of this repository and then submit a new PR (the repository is now 100MB instead of 8.8GB!). Or run the following Git commands [here](https://github.com/fontsource/fontsource/issues/456#issuecomment-1075647214), although it's a little more complicated.

Sorry for the mess! It was unfortunate you got tangled up in this!

**Comment by anuraglol** [comment]
> @kr-anurag, I'll have to apologise for interrupting your PR mid-way. This is definitely not a regular occurrence.
> 
> Just for context on what happened to your PR, #456 would explain the situation. Essentially we had to rewrite the entire Git history which is a destructive process, but it was somewhat needed for this repo. Ideally, I would've wanted to do this AFTER you submitted your PR, but I accidentally pushed to the main repository instead of a test repository 😭
> 
> To fix your PR, you could either make a **fresh clone** of this repository and then submit a new PR (the repository is now 100MB instead of 8.8GB!). Or run the following Git commands [here](https://github.com/fontsource/fontsource/issues/456#issuecomment-1075647214), although it's a little more complicated.
> 
> Sorry for the mess! It was unfortunate you got tangled up in this!

hey! sure! ill re make a pr, also nice work on the repo man! 🚀

**Pull request closed without merging**